src/pudl/io_managers.py

```diff
@@ -568,12 +568,17 @@ def __init__(self, base_path: UPath, schema: pa.Schema) -> None:
         super().__init__(base_path=base_path)
         self.schema = schema
 
-    def dump_to_path(self, context: OutputContext, obj: pd.DataFrame, path: UPath):
+    def dump_to_path(self, context: OutputContext, obj: dd.DataFrame, path: UPath):
```

Comment:
If this is really about dask dataframes, should we be calling this a `DaskParquetIOManager`?